content/docs/command-reference/repro.md

```diff
@@ -14,7 +14,8 @@ usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [targets [targets ...]]
 
 positional arguments:
-  targets        Stage or .dvc file to reproduce
+  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
+                 directories to search for stages can also be given.
```

Comment:
Perfect. We will need to submit a matching change in the core `dvc` repo. Are you able to try that too @imhardikj ? Thanks

Reply:
I checked `-p`, it doesn't work with directories. So the current is correct I think.
> submit a matching change in the core dvc repo

Sure I'll do this. Currently this is displayed for `targets`: `Stages to reproduce. 'dvc.yaml' by default.`
This should be updated to exact same text as above right: `Stage or path to dvc.yaml or .dvc file to reproduce. Using -R, directories to search for stages can also be given.`
Also this (https://github.com/iterative/dvc/blob/master/dvc/command/repro.py) is the file that needs to updated?

Reply:
Yes, https://github.com/iterative/dvc/blob/master/dvc/command/repro.py#L70-L73